sqs/sqs.go

```diff
@@ -0,0 +1,50 @@
+package sqs
+
+import (
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+
+	"github.com/pkg/errors"
+)
+
+// for testing purposes
+type SQS interface {
+	GetQueueAttributes(*sqs.GetQueueAttributesInput) (*sqs.GetQueueAttributesOutput, error)
+	// only implemented on unit tests
+	SetQueueAttributes(*sqs.SetQueueAttributesInput) (*sqs.SetQueueAttributesOutput, error)
+}
+
+type SqsClient struct {
```

Comment:
This name kinda stutters - from outside it's `sqs.SqsClient`. Also, even if `go vet` doesn't realize SQS is an acronym to complain about it, you should still make it all caps.